spec/requests/assign_controller_spec.rb

```diff
@@ -116,6 +116,15 @@
       expect(post.topic.reload.assignment.assigned_to_id).to eq(user2.id)
     end
 
+    it 'assigns topic with priority to a user' do
```

Comment:
Can we also add a test case which passes in an invalid priority like a random string or an invalid number? My hunch is that this is not handled right now and might result in us writing invalid values into the DB.

Reply:
The DB actually is smart about it because of the enums declared. Let me see if I can get you an error...

Reply:
> ArgumentError: '5' is not a valid priority

Reply:
I do not think we have any special rescues for `ArgumentError` at the controller level so this might result in a 500 response. I think we might want to ensure that the priority passed in by the client is valid and return a nicer response.

Reply:
Your concern that we're writing invalid values to the DB is addressed...

But for this one where it results in 500, I'm not sure if we have to care: the only consumer for this endpoint is our frontend, and the frontend only ever passes 1-4. It also feels a bit _off_ to test for the framework since this is a built-in enum feature.